### PR TITLE
README change

### PR DESCRIPTION
**Pull request by johnjackweir**
Let me know if I'm missing anything!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,62 @@
+# mongodb-java-demo
 This web app connects to your MongoDB Atlas cluster and uses your data to populate the different web pages. 
 It is designed to be a basic demo of how to interact with your data - so clone it, play around with it and 
 tailor it to your data!
 
-## Overview
+## Setup
+This application requires git, Java 11 and Maven 3.6. Follow the setup instructions below for the relevant OS.
+
+#####OSX
```
Comment (piakochar) [inline README.md]: [np] I think you need a space between the hashtags and the OSX

File: README.md [hunk 2]
```diff
@@ -1,9 +1,62 @@
...
+    + `brew install git`
+    + Create or login to your Github account [here](https://github.com/)
+    + Follow instructions [here](https://gist.github.com/adamjohnson/5682757) to create a new SSH key for your Github account.
+    + Restart terminal session for changes to take effect 
++ Clone this repo
+    + `cd [wherever you want the repo]`
+    + `git clone git@github.com:piakochar/mongodb-java-demo.git`
+    + OR see detailed instructions for cloning an existing repository [here](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository)
+
+#####Windows
```
Comment (piakochar) [inline README.md]: [np] same as above

**Review by piakochar — APPROVED** [review]
Awesome thanks for adding this!!

**Comment by johnjackweir** [comment]
Added spacing!

**Pull request merged**